core-primitives/types/Cargo.toml

```diff
@@ -1,16 +1,18 @@
 [package]
-name = "itp-core"
+name = "itp-types"
 version = "0.8.0"
-authors = ["bhaerdi <Bigna.Haerdi@scs.ch>"]
+authors = ["clangenbacher <christian.langenbacher@scs.ch>"]
```

Comment:
Oh, are there many places, where I am mentioned? This should probably become the hello@integritee address?

Reply:
I think we have you or @haerdib in a lot of cases. Should we open a new issue to change that?

Reply:
Ok, then let's keep this for a separate issue.

Reply:
Issue: https://github.com/integritee-network/worker/issues/393